.github/dependabot.yml

```diff
@@ -2,16 +2,75 @@
 
 version: 2
 updates:
-  # Enable version updates for Python API
+  # Python API requirements
   - package-ecosystem: pip
-    directory: "/src/bindings/python"
+    directory: "/src/bindings/python/"
     schedule:
-      interval: weekly
-      day: sunday
-      time: "13:00"
-    open-pull-requests-limit: 0
-    reviewers:
-      - jiwaszki
-      - akuporos
+      interval: "daily"
+      time: "09:00"
+      timezone: "Poland"
+    open-pull-requests-limit: 3
+    assignees:
+      - "jiwaszki"
+      - "p-wysocki"
```

Comment:
please add myself here. I would not like to miss notification from bot:)
I can update reqs files first:)

Reply:
added